src/shared/components/login/login.js

```diff
@@ -56,15 +56,20 @@ class Login extends Component {
   }
 
   render() {
-    const { error } = this.state;
+    let { error } = this.state;
+    if (error === 'Request failed with status code 401') {
```

Comment:
Hmmm could it be related to https://github.com/OperationCode/operationcode_frontend/pull/311 ? I don't think so, but the naming/timing has me worried.